src/components/CalendarDay.jsx

```diff
@@ -101,7 +104,7 @@ class CalendarDay extends React.Component {
 
     if (!day) return <td />;
 
-    const formattedDate = { date: `${day.format('dddd')}, ${day.format('LL')}` };
```

Comment:
hm, is there a reason that it was two separate format calls before?

Reply:
I'm not sure... but the result is same.